package.json

```diff
@@ -25,56 +27,50 @@
     },
     "files": [
       "node_modules/**/*",
+      "dist/main/**",
+      "dist/preload/**",
+      "dist/renderer/**",
       "package.json"
     ],
     "directories": {
       "buildResources": "assets"
     }
   },
-  "main": "public/electron.js",
   "scripts": {
-    "start": "rescripts start",
-    "build": "rescripts build",
-    "test": "rescripts test",
-    "electron-dev": "concurrently \"BROWSER=none yarn start\" \"wait-on http://localhost:3000 && electron .\"",
-    "postinstall": "electron-builder install-app-deps",
-    "pre-electron-pack": "yarn build",
-    "electron-pack": "electron-builder build --dir --windows --mac --linux"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+    "dev": "node src/scripts/watch.mjs",
+    "build": "npm run build:main & npm run build:preload & npm run build:renderer",
+    "update-vendors": "cross-env ELECTRON_RUN_AS_NODE=1 electron src/scripts/update-electron-vendors.mjs",
+    "build:main": "cd ./src/main && vite build",
+    "build:preload": "cd ./src/preload && vite build",
+    "build:renderer": "cd ./src/renderer && vite build",
+    "test": "cd ./src/renderer && vitest src",
+    "postinstall": "electron-builder install-app-deps && npm run update-vendors",
+    "pre-electron-pack": "npm run build",
+    "electron-pack": "electron-builder build --dir --windows --linux --mac"
   },
   "dependencies": {
     "@reach/accordion": "^0.8.0",
     "date-fns": "^2.9.0",
     "electron-is-dev": "^1.1.0",
-    "electron-store": "^5.1.0"
+    "electron-store": "^5.1.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@rescripts/cli": "^0.0.13",
-    "@rescripts/rescript-env": "^0.0.11",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
-    "concurrently": "^5.0.2",
-    "electron": "^7.1.10",
-    "electron-builder": "^22.3.2",
-    "node-sass": "^4.13.1",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "react-scripts": "3.4.0",
-    "wait-on": "^4.0.0"
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@types/react": "^18.0.21",
```

Comment:
Making a note for myself, probably don't need `@types` since there isn't TypeScript in this, but not blocking